version1/04 - Web services.md

```diff
@@ -1309,6 +1326,8 @@ Content-type: application/json
   - the date of creating the object
 - **userName**
   - the username accessing the object
+- **userId**
```

Comment:
The `userId` should also be added to the download log response.

Reply:
Added w/ https://github.com/documaster/noark5-web-services/pull/70/commits/5da358a44274f719cb30ba9e5c18a8e018e8f5fa